### ISSUE TITLE
Invited user via email is redirected to login url

### ISSUE THREAD

**Issue by embedded1** [joshuakarjala/django-hunger#69]
I think there is a problem in the middleware that checks if the user is authenticated and if he does not it redirects to login url.

an invited user who wasn't registered before will not be redirected to the verified url, he will be redirected to the login url instead.

then, he will have to sign up and the verified url is bypassed.

**Comment by embedded1** [comment]
Is there a reason why 'hunger.views.VerifiedView' view wasn't added to whitelisted_views list?

I just added it manually and now everything works as expected.

**Comment by yesimon** [comment]
Yep if you can add a unit test with the change then I can merge it in.

**Comment by embedded1** [comment]
So that view is missing by mistake not by itention right?
